site/src/components/Avatar/Avatar.tsx

```diff
@@ -18,20 +18,23 @@ const sizeStyles = {
   xs: {
     width: 16,
     height: 16,
-    fontSize: 8,
+    // Should never be overrided
```

Comment:
This comment should really explain why. In my experience, there's almost always a better solution than `!important`.